src/autoscaler/metricsgateway/cmd/metricsgateway/metricsgateway_test.go

```diff
@@ -70,15 +68,17 @@ var _ = Describe("Metricsgateway", func() {
 			})
 		})
 		Context("with missing configuration", func() {
+			var backupPolicyDBURL string = ""
```

Comment:
why you do backup

Reply:
@boyang9527 conf is initialized in https://github.com/cloudfoundry/app-autoscaler/blob/ca95b4ac4e5eed8adc29e63f81cec444ae839b47/src/autoscaler/metricsgateway/cmd/metricsgateway/metricsgateway_suite_test.go#L91
so it will be overwritten in this case.

Reply:
you have "missingConfig := conf" , this will not rewrite.